lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -49,23 +50,28 @@ def to_faraday_upload
       end
 
       def generate_metadata
-        veteran_full_name = @form['veteranFullName']
         address = @form['veteranAddress']
-        {
+        country_is_us = address['country'] == 'US' || address['countryCodeISO2'] == 'US'
+        veteran_full_name = @form['veteranFullName']
+        metadata = {
           'veteranFirstName' => veteran_full_name['first'],
           'veteranLastName' => veteran_full_name['last'],
           'fileNumber' => @form['vaFileNumber'] || @form['veteranSocialSecurityNumber'],
           'receiveDt' => received_date,
           # 'uuid' => "#{@uuid}_4142", # was trying to include the main claim uuid here and just append 4142
           # but central mail portal does not support that
           'uuid' => @uuid,
-          'zipCode' => address['country'] == 'USA' ? address['postalCode'] : FOREIGN_POSTALCODE,
+          'zipCode' => address['postalCode'] || address['zipCode5'],
```

Comment:
Where is `zipCode5` coming from?

Reply:
Attribute naming of a supplemental claim retrieved from the [Benefits Intake API](https://developer.va.gov/explore/api/decision-reviews/docs?version=current#:~:text=/-,supplemental_claims,-Creates%20a%20new) varies slightly from what's used on VA.gov and is required by CMP/EMMS. Instead of conditionally assigning `zipCode5` to `zipCode` in the metadata object, I added a `postalCode` attribute to the rejiggered form data [here](https://github.com/department-of-veterans-affairs/vets-api/pull/15037/files#:~:text=%27postalCode%27%20%3D%3E%20vet%5B%27address%27%5D%5B%27zipCode5%27%5D).